CourseSketchProjects/CourseSketchDatabaseServer/src/main/java/handlers/subhandlers/GradingRequestHandler.java

```diff
@@ -56,14 +56,15 @@ private GradingRequestHandler() {
      *
      * @param institution The database interface.
      * @param request The request being sent.
-     * @param authId The Id of the user who sent the request.
+     * @param authId The Id of the user who sent the request used for authentication purposes.
```

Comment:
make sure you update the doc of @param authId for all other methods that contain authId